### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by baderbuddy**
For right now the test folder is incomplete and the deployment yaml. All of the actual code changes should be ready for review though.

### CODE REVIEW

**Review by makoscafee — APPROVED** [review]
LGTM. just had few question

File: cmd/polaris/webhook.go [hunk 1]
```diff
@@ -74,79 +42,34 @@ var webhookCmd = &cobra.Command{
...
-			logrus.Debugf("Error reading namespace information: %v", err)
-		}
-
-		polarisNamespace := string(polarisNamespaceBytes)
-		if polarisNamespace == "" {
-			polarisNamespace = polarisResourceName
-			logrus.Debugf("Could not determine current namespace, creating resources in %s namespace", polarisNamespace)
+		_, err = os.Stat("/opt/cert/tls.crt")
+		if os.IsNotExist(err) {
+			time.Sleep(time.Second * 10)
```
Comment (rbren) [inline cmd/polaris/webhook.go:57]: why sleep before `panic`ing?

File: cmd/polaris/webhook.go [hunk 2]
```diff
@@ -74,79 +42,34 @@ var webhookCmd = &cobra.Command{
...
+			panic("Cert does not exist")
 		}
+		server := mgr.GetWebhookServer()
+		server.CertName = "tls.crt"
+		server.KeyName = "tls.key"
 
 		logrus.Info("Setting up webhook server")
```
Comment (rbren) [inline cmd/polaris/webhook.go:64]: I think these log messages might need some thought

File: cmd/polaris/webhook.go (hunk 1 above)
Comment (baderbuddy) [reply cmd/polaris/webhook.go:57]: Oh I forgot about that. It was to cut down on the chances of a crash loop backoff and give the job more time to update the cert.

**Comment by baderbuddy** [comment]
The ValidatingWebhookConfiguration explicitly calls out which types of objects to watch for. Polaris itself doesn't really care because it's treating them all the same anyways.

**Comment by rbren** [comment]
OK yeah I see that in the chart now - but how does that get plumbed in here?

File: .circleci/config.yml [hunk 3]
```diff
@@ -27,24 +27,24 @@ references:
...
-        curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.12.7/bin/linux/amd64/kubectl
+        curl -sLO https://storage.googleapis.com/kubernetes-release/release/v1.18.6/bin/linux/amd64/kubectl
         chmod 0755 kubectl
         sudo mv kubectl /usr/local/bin/
         kubectl version --client
 
 
         echo "Creating Kubernetes Cluster with Kind"
-        kind create cluster --wait=90s
+        kind create cluster --wait=90s --image kindest/node:v1.15.11
```
[the remark this answers is not in the document]
Comment (baderbuddy) [reply .circleci/config.yml:43]: It should work fine with other versions, but 1.16 removes a lot of deprecated APIs so we'd lose a lot in the testing if we test with 1.16 or higher.

**Comment by baderbuddy** [comment]
Either the helm install or the webhook deploy yaml. The Polaris code itself is totally agnostic.

File: cmd/polaris/webhook.go (hunk 2 above)
Comment (rbren) [reply cmd/polaris/webhook.go:64]: Mind taking a look at these? It looks like this one now comes _after_ we set up the webhook server. The message below about `listening on port` seems out of place too

Comment (baderbuddy) [reply cmd/polaris/webhook.go]: Yeah, I ripped out a lot of the log messages (I don't think they really added value), I kept the one about listening on port x, right at the end.

**Review by rbren — APPROVED** [review]

**Pull request merged**